### ISSUE TITLE
Suggestions for more additions needed

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#18]
The app essentially works on evolution chains of pokemon presently, what other cool things can we add to make this more awesome?

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
